screens/PaymentOrder.js

```diff
@@ -76,10 +76,13 @@ class PaymentOrder extends React.Component {
   handleSubmit = values => {
     var cashObj = {
       orderId: this.props.navigation.state.params.order.orderId,
-      paymentMethod: 'CASH',
+      paymentMethod: values.hasOwnProperty('cardNumber') ? 'CARD' : 'CASH',
       billType: 'SINGLE',
-      cash: values.cash
+      cash: values.cash,
+      cardType: values.cardType,
```

Comment:
only include cardType and cardNumber as payload if selected payment type is card.